### PR TITLE
Add tests for all public methods on Page

### PR DESCRIPTION
**Pull request by JoelEinbinder**
#50

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -678,6 +679,71 @@ describe('Puppeteer', function() {
...
+    it('should print to pdf', SX(async function() {
+      await page.navigate(STATIC_PREFIX + '/grid.html');
+      await page.printToPDF(outputFile);
+      expect(fs.readFileSync(outputFile).byteLength).toBeGreaterThan(0);
+    }));
+  });
+
+  describe('Page.plainText', function() {
+    it('should return the page text', SX(async function(){
+      await page.evaluate(function(){
```
Comment (aslushnikov) [inline test/test.js]: this could be done
```js
await page.setContent('<div>the result text</div>');
```

File: test/test.js [hunk 2]
```diff
@@ -678,6 +679,71 @@ describe('Puppeteer', function() {
...
+      await page.evaluateOnInitialized(function(){
+        window.injected = 123;
+      });
+      await page.navigate(STATIC_PREFIX + '/tamperable.html');
+      expect(await page.evaluate(() => window.result)).toBe(123);
+    }));
+  });
+
+  describe('Page.printToPDF', function() {
+    let outputFile = __dirname + '/assets/output.pdf';
```
Comment (aslushnikov) [inline test/test.js:718]: i wonder if pdf-generation is platform-dependent (modulo fonts?). Maybe we can just compare pdf to golden?

File: lib/Page.js [hunk 3]
```diff
@@ -285,8 +285,8 @@ class Page extends EventEmitter {
         fitWindow: false
       }),
       this._client.send('Emulation.setVisibleSize', {
-        width: width / zoom,
-        height: height / zoom,
+        width: Math.floor(width / zoom),
```
Comment (aslushnikov) [inline lib/Page.js:288]: nicee

File: test/test.js (hunk 2 above)
Comment (JoelEinbinder) [reply test/test.js:718]: the pdf files have the creation date and other meta data embedded in them. I was thinking we could render a png of the pdf with puppeteer, but I couldn't get the pdf viewer working in headless mode.

File: test/test.js (hunk 1 above)
Comment (JoelEinbinder) [reply test/test.js]: Done.

**Pull request merged**